livekit-agents/livekit/agents/voice/agent_activity.py

```diff
@@ -689,8 +689,14 @@ def _schedule_speech(
         if self.draining and not bypass_draining:
             raise RuntimeError("cannot schedule new speech, the agent is draining")
 
-        # Negate the priority to make it a max heap
-        heapq.heappush(self._speech_q, (-priority, time.monotonic_ns(), speech))
+        while True:
+            try:
+                # negate the priority to make it a max heap
+                heapq.heappush(self._speech_q, (-priority, time.perf_counter_ns(), speech))
+                break
+            except TypeError:
```

Comment:
maybe comment here on why TypeError is thrown? (it's clear when reading the bug, but that context is lost when reading the code alone)